.circleci/config.yml

```diff
@@ -1,23 +0,0 @@
-# Copyright (c) HashiCorp, Inc.
-# SPDX-License-Identifier: MPL-2.0
-
-version: 2
-jobs:
-  build:
-    docker:
-      - image: docker.mirror.hashicorp.services/cimg/go:1.17.2
-    steps:
-      - checkout
-      - run:
-          name: "Setup Environment"
-          command: |
-            echo 'export GO111MODULE=on' >> $BASH_ENV
-      - run:
-          name: "Run Tests"
-          command: make test
```

Comment:
Interesting, this wasn't running the acceptance tests right?

Reply:
Right. This would skip the acc tests, but run the unit tests. But running `make testacc` would run both.